src/subsplit_dag_storage.hpp

```diff
@@ -378,23 +378,84 @@ class GenericVerticesView {
 using VerticesView = GenericVerticesView<SubsplitDAGStorage>;
 using ConstVerticesView = GenericVerticesView<const SubsplitDAGStorage>;
 
+template <typename T>
+class PrependableVector {
+public:
+  explicit PrependableVector(PrependableVector<T>* prepend = nullptr) : prepend_{prepend ? &prepend->data_ : nullptr} {}
+
+  T& at(size_t i) {
+    if (!prepend_) return data_.at(i);
+    if (i < prepend_->size()) return (*prepend_)[i];
```

Comment:
Here we have square brackets in an `.at` implementation. Intentional?

Reply:
Yes it's intentional, because bounds check has already been done in the if() statement